ispaq/PSD_metrics.py

```diff
@@ -291,7 +291,7 @@ def do_pdf(concierge, starttime, endtime):
         try:
             resp_url = Client("IRIS")
         except Exception as e:
-            logger.error("Could not connect to 'http:/service.iris.edu/irisws/evalresp/1'")
+            logger.error("Could not connect to 'https:/service.earthscope.org/irisws/evalresp/1'")
```

Comment:
the browser will redirect, but might want to change to https://service.earthscope.org/irisws/evalresp/1 anyway